Chase https://gitlab.com/veloren/veloren/-/commit/3f7f7195cd92

error: no matching package named `gfx-backend-empty` found
location searched: https://github.com/gfx-rs/gfx?rev=27a1dae3796d33d23812f2bb8c7e3b5aea18b521
required by package `wgpu-core v0.8.0 (/wrkdirs/usr/ports/games/veloren-weekly/work/wgpu-179ea209374a92837cde252f1d9ee01f628cae08/wgpu-core)`
    ... which satisfies path dependency `wgc` of package `wgpu v0.8.0 (/wrkdirs/usr/ports/games/veloren-weekly/work/wgpu-179ea209374a92837cde252f1d9ee01f628cae08/wgpu)`
    ... which satisfies dependency `wgpu = "^0.8"` of package `egui_wgpu_backend v0.8.0 (/wrkdirs/usr/ports/games/veloren-weekly/work/egui_wgpu_backend-63a002c6a9b6c016e45806dd065864431caab621)`
    ... which satisfies git dependency `egui_wgpu_backend` of package `veloren-voxygen v0.15.0 (/wrkdirs/usr/ports/games/veloren-weekly/work/veloren-v0.15.0-403-g204834c98/voxygen)`
*** Error code 101

--- ../wgpu-179ea209374a92837cde252f1d9ee01f628cae08/wgpu-core/Cargo.toml.orig	2022-08-19 01:35:57 UTC
+++ ../wgpu-179ea209374a92837cde252f1d9ee01f628cae08/wgpu-core/Cargo.toml
@@ -39,24 +39,24 @@ gpu-descriptor = "0.1"
 gpu-alloc = "0.4"
 gpu-descriptor = "0.1"
 
-hal = { package = "gfx-hal", git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
-gfx-backend-empty = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
+hal = { package = "gfx-hal", git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
+gfx-backend-empty = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
 
 [target.'cfg(all(not(target_arch = "wasm32"), all(unix, not(target_os = "ios"), not(target_os = "macos"))))'.dependencies]
-gfx-backend-vulkan = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521", features = ["naga"] }
-gfx-backend-gl = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
+gfx-backend-vulkan = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1", features = ["naga"] }
+gfx-backend-gl = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
 
 [target.'cfg(all(not(target_arch = "wasm32"), any(target_os = "ios", target_os = "macos")))'.dependencies]
-gfx-backend-metal = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
+gfx-backend-metal = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
 #TODO: could also depend on gfx-backend-vulkan for Vulkan Portability
 
 [target.'cfg(all(not(target_arch = "wasm32"), windows))'.dependencies]
-gfx-backend-dx12 = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
-gfx-backend-dx11 = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
-gfx-backend-vulkan = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521", features = ["naga"] }
+gfx-backend-dx12 = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
+gfx-backend-dx11 = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
+gfx-backend-vulkan = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1", features = ["naga"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-gfx-backend-gl = { git = "https://github.com/gfx-rs/gfx", rev = "27a1dae3796d33d23812f2bb8c7e3b5aea18b521" }
+gfx-backend-gl = { git = "https://github.com/Imberflur/gfx", tag = "veloren-fixes-v1" }
 
 [dependencies.naga]
 git = "https://github.com/gfx-rs/naga"
